client/src/App/App.tsx

```diff
@@ -46,6 +55,8 @@ import {
 import ConnectedHomeComponent from '../containers/pages/Home/Home';
 import './App.css';
 import ConnectedSidebar from '../containers/ConnectedSidebar';
+import '@opensrp/product-catalogue/dist/index.css';
```

Comment:
Why not use this in the package itself instead of importing here?

Reply:
I figure its easier if you want to override the styles, just don't import the default ones and add your own.
several libraries out there use this, including antd itself,

Reply:
@p-netm Ah,..ok i see.. actually antd does this